packages/test/test-end-to-end-tests/src/test/benchmark/summarization.time.spec.ts

```diff
@@ -8,12 +8,13 @@ import { IContainer } from "@fluidframework/container-definitions";
 import { ContainerRuntime, DefaultSummaryConfiguration } from "@fluidframework/container-runtime";
 import { channelsTreeName } from "@fluidframework/runtime-definitions";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
-import { TelemetryNullLogger } from "@fluidframework/telemetry-utils";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
 import { ITestContainerConfig, ITestObjectProvider } from "@fluidframework/test-utils";
 import { describeNoCompat, ITestDataObject } from "@fluidframework/test-version-utils";
 import { benchmark } from "@fluid-tools/benchmark";
 import { ISummaryBlob, SummaryType } from "@fluidframework/protocol-definitions";
-import { bufferToString } from "@fluidframework/common-utils";
+import { bufferToString, TelemetryNullLogger } from "@fluidframework/common-utils";
```

Comment:
`TelemetryNullLogger` was deprecated in `common-utils` in favor of the one in `telemetry-utils`, I think we should go back to using that one (here and in `summarization.memory.spec.ts`).

Reply:
Did not know about this! Updated